src/components/PriceAlertForm/PriceAlertForm.tsx

```
@@ -1,6 +1,8 @@
 'use client';
 import React, { useState, useEffect, useRef, useLayoutEffect } from 'react';
 import { useForm, FormProvider } from 'react-hook-form';
+import { ConfirmDialog } from '../ui/confirm-dialog';
+import { ConfirmDialog } from '../ui/confirm-dialog';
```

Biome (2.1.2)

[error] Shouldn't redeclare 'ConfirmDialog'. Consider to delete it or rename it.

'ConfirmDialog' is defined here:


(lint/suspicious/noRedeclare)